src/main/java/org/cbio/gdcpipeline/GDCPipelineApplication.java

```diff
@@ -0,0 +1,88 @@
+package org.cbio.gdcpipeline;
+
+import org.apache.commons.cli.*;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersBuilder;
+import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing;
+import org.springframework.batch.core.launch.JobLauncher;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+
+
+@SpringBootApplication
+@EnableBatchProcessing
+public class GDCPipelineApplication {
+
+
+	private static Log LOG = LogFactory.getLog(GDCPipelineApplication.class);
+
+	private static Options getOptions(String []input){
+		Options options = new Options();
+		options.addOption("j","job",false,"Job to run");
+        options.addOption("study", "study", true, "Cancer Study Id");
+        options.addOption("s","source",true,"source directory for files");
+		options.addOption("o","output",true,"output directory for files");
+		options.addOption("h", "help", false, "shows this help document and quits.");
+		return options;
+	}
+
+	private static void help(Options gnuOptions, int exitStatus) {
+		HelpFormatter helpFormatter = new HelpFormatter();
+		helpFormatter.printHelp("GDCPipelineApplication", gnuOptions);
+		System.exit(exitStatus);
+	}
+
+    private static void launchJob(String[] args, String sourceDirectory, String outputDirectory, String jobName, String study) throws Exception {
+        SpringApplication app = new SpringApplication(GDCPipelineApplication.class);
+		ApplicationContext ctx= app.run(args);
+		Job mainJob = ctx.getBean("mainJob", Job.class);
+		long t = System.currentTimeMillis();
+		String time = String.valueOf(t);
```

Comment:
This is temp solution. I face this issue while re-running the Batch that it doesn't allow multiple runs of the same job. Adding time just makes every job unique while running so it doesnt clash with job repository info.
I have a TODO to check on this again.